samcli/yamlhelper.py

```diff
@@ -23,6 +23,15 @@
 import yaml
 from yaml.resolver import ScalarNode, SequenceNode
 
+TAG_STR = "tag:yaml.org,2002:str"
+
+
+def string_representer(dumper, value):
```

Comment:
could you add a docstring here, just to explain what happens here.

Reply:
Good Callout, I meant to go back and completely forgot. Will get a new revision with that here soon.